README.md

```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there is only a few points you need to check if you have issue with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the SBUS bridge)
+
+1.  You are using an "upboard" kernel. Check for that using `uname -a`
```

Comment:
How is one supposed to see from this output if he runs an upboard kernel?

Reply:
done